js/base/Exchange.js

```diff
@@ -670,18 +670,20 @@ module.exports = class Exchange {
     }
 
     filterByValueSinceLimit (array, field, value = undefined, since = undefined, limit = undefined, key = 'timestamp', tail = false) {
-        const valueIsDefined = value !== undefined && value !== null
-        const sinceIsDefined = since !== undefined && since !== null
+        const firstEntry = this.safeValue (array, 0);
+        const valueIsDefined = value !== undefined && value !== null;
+        const sinceIsDefined = since !== undefined && since !== null && firstEntry[key] !== undefined; // Check if user has defined since, and check if timestamp key is actually in the response
```

Comment:
I'm not really sure if this is the correct solution.

The only timestamp information that ndax returns is for `LastUpdateTimestamp` and it has a really weird format where if it's read as nanoseconds the date value would be "Wed Mar 14 16:14:51 UTC 1990" while if it were microseconds the date would be "Tue May 23 23:34:21 UTC 22169", both of which are impossible, so right now the timestamp is undefined for every entry.

I've been trying to open a ticket with Ndax, but receive `unexpected server error` every time I click submit ticket.

@kroitor @frosty00 @carlosmiei Please give your input to this solution

Reply:
@samgermain yeah it seems an invalid timestamp, so I'd also keep it undefined

Reply:
> @samgermain yeah it seems an invalid timestamp, so I'd also keep it undefined

With it being undefined, it means the response is empty, because we filter by time, but with that response added to the filter method, then it isn't undefined

Reply:
@samgermain I do not understand this change tbh; if the timestamp provided is invalid and we let it be `undefined` within our unified structure `filterByCurrencySinceLimit`will just ignore it, what am I missing here?

Reply:
> 

It's not ignored, when the user gives a timestamp, but the response object contains no timestamps, then every record in the response is filtered out